models/omw.ini

```diff
@@ -0,0 +1,11 @@
+[model]
+name = omw
```

Comment:
For the curious, this is common leet-speak for "on my way".  I originally called it `under-construction` but I ran into a problem.  I changed it to a single word with no separator to eliminate that as a possibility and discovered that wasn't the source of the problem.

I just never changed it back.  This site isn't supposed to live very long so... whatever.  It's not customer-facing.